src/py42/services/detectionlists/user_profile.py

```diff
@@ -171,7 +172,12 @@ def add_cloud_alias(self, user_id, alias):
             u"cloudUsernames": [alias],
         }
         uri = self._make_uri(u"/addcloudusernames")
-        return self._connection.post(uri, json=data)
+        try:
+            return self._connection.post(uri, json=data)
+        except Py42BadRequestError as err:
+            if "Cloud usernames must be less than or equal to 2" in err.response.text:
```

Comment:
can we remove "2" from the end of this message to protect against the possibility that this limit could be changed? That way this exception will still trigger if we get `Cloud username must be less than or equal to 5`, for example.